docs/advanced/node-local-storage.md

```diff
@@ -0,0 +1,356 @@
+# Using node local storage on kubernetes
+
+This guide shows different ways to use node local on Lokomotive, to achieve near
+bare-metal performance inside Kubernetes pods.
+
+Before you continue, make sure to read generalities about these concepts:
+ * Volumes: https://kubernetes.io/docs/concepts/storage/volumes/
+ * Persistent Volumes and PVC: https://kubernetes.io/docs/concepts/storage/persistent-volumes/
+ * StorageClasses: https://kubernetes.io/docs/concepts/storage/storage-classes/
+
+More detail about specific volumes used and how to use them on lokomotive will
+follow, so no need to look into volume specifics on the first read.
+
+**NOTE**: This has been tested on Packet, some lokomotive changes need to be
+ported for this to work on other platforms.
+
+## Using local persistent volume: filesystem mode
+
+This way of using node local storage works by exposing a filesystem mounted in
+the host as a persistent volume to pods.
+
+One persistent volume can be bound to only one PVC, so only one pod will be able
+to use one persistent volume in the host. However, see the section(link) bellow
```

Comment:
```suggestion
to use one persistent volume in the host. However, see the section(link) below
```

I think "link" might be a placeholder for anchor to section below. Still needs to be added. Or just remove it and have `"section below".

Reply:
Removed, it was a placeholder but removing is better. Thanks!